.eslintrc

```diff
@@ -2,8 +2,10 @@
   "env": {
     "node": true
   },
-  "extends": "airbnb",
+  "extends": "airbnb-base",
   "rules": {
+    "import/no-mutable-exports": 0,
+    "global-require": 0,
```

Comment:
Test mocks currently export mutable variables, `let stdout` and `let stderr`.  I turned off `import/no-mutable-exports` to allow this until the mocks are reworked per #46 comments.

`test/specs/cli.js` has a `require()` in a try/catch, forbidden by `global-require`.  Turned this rule off as well.  It demands all `require()` calls be at the top of the file.  Not sure we'll ever want this rule on.